x/campaign/types/params_test.go

```diff
@@ -82,3 +87,37 @@ func TestValidateTotalSupplyRange(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateCampaignCreationFee(t *testing.T) {
+	tests := []struct {
+		name        string
+		creationFee interface{}
+		err         error
+	}{
+		{
+			name:        "invalid interface",
+			creationFee: "test",
+			err:         fmt.Errorf("invalid parameter type: string"),
+		},
+		{
+			name:        "invalid coin",
+			creationFee: sdk.Coins{sdk.Coin{Denom: "foo", Amount: sdk.NewInt(-1)}},
+			err:         errors.New("coin -1foo amount is not positive"),
+		},
+		{
```

Comment:
If `DefaultCampaignCreationFee` is empty coins, we should also have a test case with `sample.Coins()`